src/diffusers/models/vae.py

```diff
@@ -424,6 +424,7 @@ def __init__(
         num_vq_embeddings: int = 256,
         norm_num_groups: int = 32,
         vq_embed_dim: Optional[int] = None,
+        scaling_factor: int = 0.18215,
```

Comment:
Maybe add a comment to the docstrings above, similar to what @Ghost---Shadow suggested in #1828: https://github.com/huggingface/diffusers/pull/1828/files

Reply:
If helpful in docstring, originally documented in LDM sections 4.3.2 and D.1 https://arxiv.org/abs/2112.10752